docs/og-analytics/worked-out-examples/worked-out-example-stir-futures-option-normal-moneyness.rst

```diff
@@ -0,0 +1,112 @@
+==========================================
+
+The code of this example can be found in the test: **STIRFuturesOptionNormalExpSimpleMoneynessMethodE2ETest**.  
+Other examples are available in **STIRFuturesOptionNormalExpSimpleMoneynessGBPE2ETest** focusing on GBP-dominated trades and specific outputs.  
+Our discussion in this document follows the former test, but is transferable to the latter.
+
+Curves
+------
+
+See :doc:`../worked-out-example-stir-futures-discounting`.
+
+Parameter provider
+------------------
+
+Given a volatility surface in the form of **InterpolatedDoublesSurface** where the volatility surface is parametrised by option's expiry and simple moneyness, the parameter provider for option valuation based on normal model is created by::
+
+    NormalSTIRFuturesExpSimpleMoneynessProviderDiscount NORMAL_MULTICURVES = 
+      new NormalSTIRFuturesExpSimpleMoneynessProviderDiscount(MULTICURVES, VOL_SURFACE_SIMPLEMONEY, GBPLIBOR3M, false);
+
+Here the simple moneyness is given by the underlying futures price minus strike price for **false** input. 
+
+Instrument
+----------
+First we create an underlying STIR futures security with data stored in the example::
+
+    InterestRateFutureSecurityDefinition ERU2_DEFINITION = 
+      new InterestRateFutureSecurityDefinition(LAST_TRADING_DATE, EURIBOR3M, NOTIONAL, FUTURE_FACTOR, NAME, TARGET);
+
+Then we crate an option on the STIR futures::
+
+    InterestRateFutureOptionMarginSecurityDefinition OPTION_ERU2_DEFINITION = 
+      new InterestRateFutureOptionMarginSecurityDefinition(ERU2_DEFINITION, EXPIRATION_DATE, STRIKE, IS_CALL);
+
+We create a transaction on the above security with data stored in the file::
+
+    InterestRateFutureOptionMarginTransactionDefinition TRANSACTION_1_DEFINITION = 
+      new InterestRateFutureOptionMarginTransactionDefinition(OPTION_ERU2_DEFINITION, QUANTITY, TRADE_DATE_1, TRADE_PRICE);
+
+The transaction is the purchase/sell of a given quantity on a given date for a given price.
+ 
+This is the definition version of the transaction, i.e. it contains all the dates and data relevant to the trade.
+
+The definition version is converted to the *derivative* version::
+
+    InterestRateFutureOptionMarginTransaction TRANSACTION_1 = TRANSACTION_1_DEFINITION.toDerivative(REFERENCE_DATE, MARGIN_PRICE);
+
+In this transformation, the dates are replaced by time between the valuation date and the different cash flow dates. Moreover the last priced used in the margining is stored. This is used to compute the present value which is the difference between the current price and the last margin price.
+
+
+In this examples all the computation are done with a *projection and discounting* model, that we simply call *discounting*. No convexity adjustment for the daily margining is computed in this implementation. Another worked out example deal with convexity adjustment; **STIRFuturesTransactionHullWhiteMethodE2ETest**.
```

Comment:
The reference on HW convexity adjustment is for STIR futures, not options. We don't have a documentation on STIR futures options with HW yet.